at-chuongphu/excercise6.rb

```diff
@@ -0,0 +1,31 @@
+cong = proc { |*ar| ar.inject {|a, b| a + b} }
+tru = proc { |*ar| ar.inject {|a, b| a - b} }
+nhan = proc { |*ar| ar.inject {|a, b| a * b} }
+chia = proc do |*ar|
+  ar.inject { |a, b| a / b unless b.zero? }
+end
+p cong.(3,1,4)
+p tru.(7,2)
+p nhan.(5,2)
+p chia.(100,5,2)
+# ex2
+def add_more(num)
+  proc { |x| num + x }
+end
+
+def multi_more(num)
+  proc { |x| num * x }
+end
+
+def compose(proc1, proc2)
+  proc { |x| proc2.(proc1.(x)) }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/2528ef3bd0109a08bff8a72dd8a5fb8146369348/at-chuongphu/excercise6.rb#L21"></span>Style/LambdaCall: Prefer the use of <code>lambda.call(...)</code> over <code>lambda.(...)</code>. (https://github.com/rubocop-hq/ruby-style-guide#proc-call)</td>
    </tr>
  </tbody>
</table>